2018-04-26 02:59:01.408896
Traceback (most recent call last):
  File "printChecker_2.py", line 8, in <module>
    check = checker2.check()
NameError: name 'checker2' is not defined
2018-04-26 04:15:01.518530
Traceback (most recent call last):
  File "printChecker_2.py", line 8, in <module>
    check = checker2.check()
NameError: name 'checker2' is not defined
2018-04-26 09:45:01.903158
Traceback (most recent call last):
  File "printChecker_2.py", line 8, in <module>
    check = checker2.check()
NameError: name 'checker2' is not defined
